types/materializecss__materialize/package.json

```
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "cash-dom": "*"
```

```suggestion
        "cash-dom": "*"
```
this is no-n-, this should not be pass through versions, please use same from package or pick current ono (recall recent versioning dramas on npm)

---

Added major version requirement in last commit mauromascarenhas@ec93c3b.